sql/_27_banana_qa/issue_12401_DML_Natural_Join/cases/03_natural_join_bug_from_web_001.sql

```diff
@@ -52,7 +52,7 @@ select * from a,b where a.id=b.id and a.name=b.name order by 1,2;
 
 select count(*) from a natural join b;
 
-select * from a  minus  order by 1,2 select * from b order by 1,2;
+select * from a  minus  order by 1,2; select * from b order by 1,2;
```

Comment:
This was often the case, but it would be better to use a newline based on the delimiter.

Reply:
The review has been reflected.